### ISSUE TITLE
surf_me only has one of the bonuses zoned instead of the main map

### ISSUE THREAD

**Issue by bonjorno7** [dPexxIAM/Momentum-surf-zones#2]
also other stage maps only have start and end zones but no stage zones which doesn't matter cause you can't get stage records right now but ought to be fixed eventually

**Comment by dPexxIAM** [comment]
Yeah majority of these zones were made by wayne a while back, he did say he planned to go back and re-zone the maps that were zoned poorly, however i doubt we will see that happen now.

Are bonus zones available now? Sadly i am on 0.6.2 so i am not sure if  they are included but if you can remember which stage maps are zoned poorly i can fix those.
